src/neml2/drivers/solid_mechanics/SolidMechanicsDriver.cxx

```diff
@@ -46,29 +46,36 @@ SolidMechanicsDriver::expected_options()
   options.set("cauchy_stress").doc() = "Cauchy stress";
 
   options.set<VariableName>("temperature") = VariableName("forces", "T");
-  options.set("temperature").doc() = "Temperature";
-
+  options.set<VariableName>("fixed_values") = VariableName("forces", "fixed_values");
   options.set<CrossRef<torch::Tensor>>("prescribed_strains");
   options.set("prescribed_strains").doc() = "Prescribed strain (when control = STRAIN)";
 
   options.set<CrossRef<torch::Tensor>>("prescribed_stresses");
   options.set("prescribed_stresses").doc() = "Prescribed stress (when control = STRESS)";
 
   options.set<CrossRef<torch::Tensor>>("prescribed_temperatures");
-  options.set("prescribed_temperatures").doc() = "Prescribed temperature";
+  options.set<CrossRef<torch::Tensor>>("prescribed_mixed_conditions");
+
+  options.set<VariableName>("control_name") = VariableName("forces", "control");
+  options.set<CrossRef<torch::Tensor>>("prescribed_control");
```

Comment:
Also to blame auto merging.

Reply:
Some option docstrings will definitely help, I think I'm already confusing `prescribed_mixed_conditions` with `prescribed_control`.